docs/agents/sematext-agent/events.md

```diff
@@ -0,0 +1,63 @@
+title: Sematext Agents Events
+
+Below you can find a list of all the events sent from Sematext Agent
```

Comment:
Should we provide link(s) to where in SC Infra one can see these events?
Should we point out Events heatbar in Monitoring?
Should we mention Split Screen with Events view on one side?